services/user.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(id=user_id)
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None,
+) -> User:
+    return get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email if email else "",
+        first_name=first_name if first_name else "",
+        last_name=last_name if last_name else "",

```

Comment:
The `email` field can be `None`, the other two expressions may be simplified using `or` operators. 
Alternatively, it's possible to set default names to those arguments to `""` even though it contradicts instructions. :)